### ISSUE TITLE
Ajax request fired two times when using "responsive:true"

### ISSUE THREAD

**Issue by tiipiik** [yajra/laravel-datatables#1608]
I was having a problem of ajax request fired two times on page load and don't know why, but I finally find.
I'm using server side processing, and if I use "responsive:true" the ajax request is fired two times. If i use "responsive:false" or remove "responsive" everything is fine and the resquest is fired only once.

```javascript

        var vDataTable = $('#second-data-table');
        var datatable_url = vDataTable.data('url');
        var $filterTag = $('.filter-tag');
        var dataTable = vDataTable.DataTable({
            stateSave: true,
            responsive: true, // If set to true the ajax request is fired two times
            processing: true,
            serverSide: true,
            deferRender: true,
            ajax: {
                url: datatable_url,
                data: function (d) {
                    var tag = $filterTag.val();
                    if (tag) {
                        d.tag = tag;
                    }
                }
            },
            columns: [
                { data: 'id', name: 'id' }
            ],
        });
```

- Operating System
- PHP Version : 7.1
- Laravel Version : 5.5
- Laravel-Datatables Version : lastest

**Comment by yajra** [comment]
Not sure on this one but I think this is on the javascript level and is out of scope of the package. I suggest you ask / search on DataTables.net forum. Thanks!

**Comment by tiipiik** [comment]
Oh yeah my bad, you're totally right !
I'll submit this to them.
